nodes/core/io/21-httpin.js

```diff
@@ -232,6 +250,15 @@ module.exports = function(RED) {
                     msg.payload += chunk;
                 });
                 res.on('end',function() {
+                    if(RED.settings.metricsOn && RED.settings.metricsOn === true) {
+                        if(preRequestTimestamp) {
+                            var elapsedTimeDiff = process.hrtime(preRequestTimestamp);
+                            node.metric(msg, "request.duration.nanos", (elapsedTimeDiff[0] * 1e9 + elapsedTimeDiff[1]));
+                        }
+                        if(res.client && res.client.bytesRead) {
+                            node.metric(msg, "response.client.bytes.read", res.client.bytesRead);
```

Comment:
@knolleary Is this what we need or shall I calculate the size based on msg.payload similarly to how it has been calculated for the content-length above?